src/searching.md

```diff
@@ -132,15 +132,18 @@ finds notes with no tags.
 finds notes with tags starting with "ani".
 
 `deck:french`\
-find cards in a deck called "French", or its subdecks like "French::Vocab".
+find cards in a top-level deck called "French", or its subdecks like "French::Vocab". It will not match subdecks with that name, such as "Languages::French".
```

Comment:
```suggestion
find cards in a top-level deck called "French", or its subdecks like "French::Words". It will not match subdecks with that name, such as "Languages::French".
```